helm/sematic-server/templates/configmap.yaml

```diff
@@ -15,3 +15,4 @@ data:
 {{ end }}
   KUBERNETES_NAMESPACE: {{ .Release.Namespace }}
   SEMATIC_WORKER_KUBERNETES_SA: {{ .Values.worker.service_account.name | quote }}
+  SEMATIC_WORKER_API_ADDRESS: "http://{{ include "sematic-server.fullname" . }}"
```

Comment:
This is just saying: "populate this config from the k8s service name," which works because k8s in-cluster DNS resolves services by their names.

Reply:
what about https support, and it being optional?